lib/cgroup/cgroup_test.go

```diff
@@ -27,107 +27,89 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 
 	"github.com/google/uuid"
-	"gopkg.in/check.v1"
+	"github.com/stretchr/testify/require"
 )
 
 func TestMain(m *testing.M) {
 	utils.InitLoggerForTests()
 	os.Exit(m.Run())
 }
 
-type Suite struct{}
-
-var _ = check.Suite(&Suite{})
-
-func TestControlGroups(t *testing.T) { check.TestingT(t) }
-
-// TestCreate tests creating and removing cgroups as well as shutting down
+// TestRootCreate tests creating and removing cgroups as well as shutting down
 // the service and unmounting the cgroup hierarchy.
-func (s *Suite) TestCreate(c *check.C) {
+func TestRootCreate(t *testing.T) {
+	t.Parallel()
```

Comment:
We should move these below the `isRoot` checks so we don't stack up a bunch of parallelized tests that will immediately get skipped afterwards when not running as the superuser.